client/landing/gutenboarding/stores/onboard/reducer.ts

```diff
@@ -43,7 +43,24 @@ const siteVertical: Reducer<
 	return state;
 };
 
-const reducer = combineReducers( { domain, siteTitle, siteVertical } );
+const pageLayouts: Reducer< Set< string >, ReturnType< typeof Actions[ 'togglePageLayout' ] > > = (
+	state = new Set(),
```

Comment:
I suspect we'll end up persisting this, `Set` won't serialize well, and we don't have fine-grained control over the wp-data persistence behavior.

Shall we add `pageLayouts` to the persistence array and see how this behaves?

Reply:
Good catch. It should work now with `Array`. I've set it to persist and we can remove later if needed.

Reply:
+ good to know :-)